lib/core/services/log.service.ts

```diff
@@ -34,82 +45,79 @@ export class LogService {
         }
     }
 
-    get error(): (message?: any, ...optionalParams: any[]) => any {
+    error(message?: any, ...optionalParams: any[]) {
         if (this.currentLogLevel >= LogLevelsEnum.ERROR) {
-            return console.error.bind(console);
+
+            this.messageBus(message, 'ERROR');
+
+            console.error(message, ...optionalParams);
```

Comment:
The whole point and complexity of the original code was to preserve stack trace in the debugger, and point debugger to the code that raise the error, rather than log service itself.  The change pretty much kills the purpose.

Reply:
It was calling the log as well before binding it . Anyway I didn't find a better solution to put between the bind and the call the message bus

Reply:
That is fine, was just explaining the previous implementation and why it’s ugly :)